stratisd_nonrust.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -e
+
+export PATH="$HOME/.cargo/bin:$PATH"
+export STRATIS_DEPS_DIR=$WORKSPACE/stratis-deps
+export RUST_LOG=libstratis=debug
+export TEST_BLOCKDEVS_FILE=~/test_config.json
```

Comment:
Should omit ```TEST_BLOCKDEVS_FILE```, it has become "dead" in this script.

Reply:
OK; I can remove it.